mbuild/formats/lammpsdata.py

```diff
@@ -856,29 +550,85 @@ def write_lammpsdata(
                         )
                     )
 
+    return
+
+
+def _evaluate_lj_conversion_factors(
+    structure, conversion_name, conversion_factor
+):
+    """Get Lennard Jones style conversion factors. `conversion_name`` can be sigma, epsilon, or mass."""
+    if conversion_factor is None:
+        # Check if structure is parametrized
+        if any([atom.sigma for atom in structure.atoms]) is None:
+            raise ValueError(
+                "LJ units specified but one or more atoms has undefined LJ "
+                "parameters."
+            )
+        else:
+            conversion_factor = np.max(
+                [getattr(atom, conversion_name) for atom in structure.atoms]
+            )
+            warn(
+                f"Assuming {conversion_name} conversion factor of "
+                + str(conversion_factor)
+            )
+        if conversion_factor == 0:
+            conversion_factor = 1
+            warn(
+                f"{conversion_name} conversion factor cannot be inferred from the maximum {conversion_name} value in the ParmEd Structure. "
```

Comment:
If the `conversion factor` starts as 0 and later set to 1 by default, both warnings in line 571 and 578 will be printed (first `assuming 0` and then `cannot be inferred; set to 1`.) Makes sense but looks a little weird to have two warnings. maybe move the warning in 571 to another if statement below (`if conversion_factor > 0...`)

Reply:
Good catch, I'll fix this up so we don't get two warnings.

Reply:
This is actually in an if elif statement, so they can't both get warned!